### PR TITLE
Re-make of Departmental Skirts: Engineering

### PR DESCRIPTION
**Pull request by CydiaLamiales**
## About The Pull Request
Essentially a remake of https://github.com/BeeStation/BeeStation-Hornet/pull/6075
Adds suitskirts for Chief Engineer, Station Engineer and Atmospheric Technician. 
All sprites were created by https://github.com/Naevii

## Why It's Good For The Game
More customization good and engineering is the only department without jumpskirts.

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

CE's outfit box

image

image

image

image

Wardrobes and compression boxes : 

image

image

image

image

</details>

##
:cl:CydiaLamiales - Code, Naevi - Sprites
add: Added Chief Engineer, Station Engineer and Atmospheric Technician jumpskirts
imageadd: Added icons for engineering department suitskirts
/:cl:

### CODE REVIEW

**Comment by TeomanTheGreat** [comment]
OSHA has been violated. Truly despicable NT budget cuts (I can only assume a invisible layer of lead blocks the legs with this thing because it would make less sense for the rad resist the suit has)

**Review by PestoVerde322 — DISMISSED** [review]
Icons looks fine, Code also looks fine, Approved, and OSHA cannot operate on Space turf.

**Comment by itsmeow** [comment]
They look fantastic. Finally, I can live out my OSHA violating dreams

**Review by itsmeow — DISMISSED** [review]

**Comment by PowerfulBacon** [comment]
If you pass the player vote I'll merge, if you fail the player vote I'll close

**Comment by Effemu** [comment]
Skirts have no place in engineering, NT isn't there to protect your feelings, but to make production and research more efficient. Allowing employees to wear skirts in engineering department is the opposite of efficient, creating additional costs for NT and workplace hazards which lowers profits of the corporation. If you want skirts in engineering go play LRP server where they belong.

🗿

**Comment by Archanial** [comment]
it doesn't make sense for an engineer to wear a skirt
prove me wrong I will wait

**Comment by CydiaLamiales** [comment]
> it doesn't make sense for an engineer to wear a skirt prove me wrong I will wait

Doesn't really make sense for security either, or virology. And yet, here we are.

**Comment by Archanial** [comment]
You are welcome to fix the mistake by opening pr that removes them.
This is not a counter argument.

File: code/game/objects/structures/crates_lockers/closets/job_closets.dm [hunk 1]
```diff
@@ -156,6 +156,7 @@
 		/obj/item/storage/backpack/industrial = 1,
 		/obj/item/clothing/suit/hooded/wintercoat/engineering/atmos = 3,
 		/obj/item/clothing/under/rank/engineering/atmospheric_technician = 3,
+		/obj/item/clothing/under/rank/engineering/atmospheric_technician/skirt = 3,
```
Comment (Archanial) [inline code/game/objects/structures/crates_lockers/closets/job_closets.dm]: bloat, if anything move this to some vendor or make it spawn less stuff overall
We already have too much stuff in locker anyways and I feel like this should be niche option or just be in vendor for free

**Review by Archanial — DISMISSED** [review]
comment above

File: code/game/objects/structures/crates_lockers/closets/job_closets.dm (hunk 1 above)
Comment (CydiaLamiales) [reply code/game/objects/structures/crates_lockers/closets/job_closets.dm]: This wardrobe locker literally doesn't spawn anywhere btw.

Comment (CydiaLamiales) [reply code/game/objects/structures/crates_lockers/closets/job_closets.dm]: And the engineering one only appears on Box in one instance, I'd say it's a unique map feature instead.

**Comment by TeomanTheGreat** [comment]
This can all be solved if you just remove some of the rad resist on the skirt version since the legs aren't covered. I have saved beestation. I will be accepting my gold medal now.

**Comment by CydiaLamiales** [comment]
> This can all be solved if you just remove some of the rad resist on the skirt version since the legs aren't covered. I have saved beestation. I will be accepting my gold medal now.

It only has like, 10 rad resist, barely anything. I could bump it a bit lower for flavor but I fear it might break the armor listings.

**Comment by TeomanTheGreat** [comment]
> > This can all be solved if you just remove some of the rad resist on the skirt version since the legs aren't covered. I have saved beestation. I will be accepting my gold medal now.
> 
> It only has like, 10 rad resist, barely anything. I could bump it a bit lower for flavor but I fear it might break the armor listings.

It would make it make a lot more sense though. If you pick the skirt you have to deal with less protection from rads.

File: code/game/objects/structures/crates_lockers/closets/job_closets.dm (hunk 1 above)
Comment (CydiaLamiales) [reply code/game/objects/structures/crates_lockers/closets/job_closets.dm]: Moved to a compression box following the convo in Discord.

**Comment by CydiaLamiales** [comment]
> > > This can all be solved if you just remove some of the rad resist on the skirt version since the legs aren't covered. I have saved beestation. I will be accepting my gold medal now.
> > 
> > 
> > It only has like, 10 rad resist, barely anything. I could bump it a bit lower for flavor but I fear it might break the armor listings.
> 
> It would make it make a lot more sense though. If you pick the skirt you have to deal with less protection from rads.

image
There, now it has 7 rad resist instead of 10

**Comment by PowerfulBacon** [comment]
Based

**Comment by Pirill** [comment]
Did you actually just argue for a grudge stat change over a cosmetic? Holy fuck this community is something else.

**Comment by bluezorua** [comment]
oh dear god not an engineering skirt, how will I be immersed in engineering while the 6 foot tall lizard and moth work on a yellow crystal that makes energy out of nothing and a purple skeleton and robot re-create the sun 15 metres away from us

**Comment by Crossedfall** [comment]
Just to better clarify my stance, I've got nothing against the jump skirts. I just don't like them being a standard outfit choice for certain roles. I'd far prefer them to be special outfits for more formal occasions when they're not at work. Something like security's old formal wear.

I've always kind of preferred when clothing matches the job and the area they work in. I find it significantly more appealing and I think it adds a lot more to the setting. I also realize that we've drifted from that a bit, which is why I'm not taking such a hard stance on this.

**Review by PestoVerde322 — APPROVED** [review]
Democracy won, code is ok, icons are ok, approved.

**Comment by CydiaLamiales** [comment]
WOOOOOOO YEAH 🎉

**Pull request merged**